### DISCUSSION TITLE
Welcome to gp3s Discussions!

### DISCUSSION THREAD

**Discussion by djong1** [djong1/gp3s#32] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  We will use this community to discuss improvements and use issues to log identified issues that have a clear ask and outcome.

**Comment by nlars2468** [comment]
Hi !
 When looking for Evaluated Ranking by Spot for 2023 no data is available, but whem 
searching for 2022 or earlier data show up.

Is there any reasoon for this ?

Lars N
SWE-170
